### PR TITLE
Fix en api doc of shape, scale, split, selu, soft_relu, scatter

### PR DESCRIPTION
**Pull request by cryoco**
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html (1)
Fix en api doc of shape, scale, split, selu, soft_relu, scatter
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html (3)
10 162 69 147_8089_documentation_docs_en_beginners_guide_index_en html
10 162 69 147_8089_documentation_docs_en_api_layers_selu html

### CODE REVIEW

**Review by Superjomn — DISMISSED** [review]
LGTM

**Review by Superjomn — DISMISSED** [review]
LGTM

File: python/paddle/fluid/layers/nn.py [hunk 1]
```diff
@@ -6185,25 +6185,24 @@ def reduce_any(input, dim=None, keep_dim=False, name=None):
...
     """
-    Split the input tensor into multiple sub-tensors.
+    Split the input tensor into multiple sub-Tensors.
 
     Args:
-        input (Variable): The input variable which is a Tensor or LoDTensor.
-        num_or_sections (int|list): If :attr:`num_or_sections` is an integer,
-            then the integer indicates the number of equal sized sub-tensors
-            that the tensor will be divided into. If :attr:`num_or_sections`
+        input (Variable): The input variable which is an N-D Tensor or LoDTensor, dtype being float32, float64, int32 or int64.
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: dtype -> data type

File: python/paddle/fluid/layers/nn.py [hunk 2]
```diff
@@ -9894,39 +9893,69 @@ def scatter(input, index, updates, name=None, overwrite=True):
...
 
     Args:
-        input (Variable): The source input with rank>=1.
-        index (Variable): The index input with rank=1. Its dtype should be
-                          int32 or int64 as it is used as indexes.
-        updates (Variable): The updated value of scatter op.
-        name (str|None): The output variable name. Default None.
-        overwrite (bool): The mode that updating the output when has same index.
+        input (Variable): The input N-D Tensor with rank>=1. Data type can be float32.
+        index (Variable): The index 1-D Tensor. Data type can be int32, int64.
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: input 和 index 的shape应该有关联限制之类的吧

File: python/paddle/fluid/layers/nn.py [hunk 3]
```diff
@@ -12541,29 +12609,49 @@ def _elementwise_op(helper):
...
+        x(Variable): Input N-D Tensor of scale operator. Data type can be float32, float64, int8, int16, int32, int64, uint8.
+        scale(float): The scale factor of the input.
+        bias(float): The bias to be put on the input.
+        bias_after_scale(bool): Apply bias addition after or before scaling. It is useful for numeric stability in some circumstances.
+        act(str, optional): Activation applied to the output such as tanh, softmax, sigmoid, relu.
+        name(str, optional): The default value is None. Normally there is no need for user to set this property.  For more information, please refer to :ref:`api_guide_Name` 
 
     Returns:
-        out(${out_type}): ${out_comment}
+        Variable(Tensor|LoDTensor): Output tensor of scale operator.
```
Comment (heavengate) [inline python/paddle/fluid/layers/nn.py]: data type 说明一下

**Comment by cryoco** [comment]
@heavengate Done. Thanks!

**Review by Superjomn — DISMISSED** [review]
LGTM

**Review by heavengate — DISMISSED** [review]

**Review by Superjomn — DISMISSED** [review]

**Review by heavengate — DISMISSED** [review]

**Review by Superjomn — DISMISSED** [review]

**Review by heavengate — DISMISSED** [review]

**Review by Superjomn — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]
LGTM

**Pull request merged**